authenticate/authenticate.go

```diff
@@ -40,9 +40,6 @@ func ValidateOptions(o *config.Options) error {
 	if o.AuthenticateCallbackPath == "" {
 		return errors.New("authenticate: 'AUTHENTICATE_CALLBACK_PATH' is required")
 	}
-	if err := urlutil.ValidateURL(o.DataBrokerURL); err != nil {
-		return fmt.Errorf("authenticate: invalid 'DATABROKER_SERVICE_URL': %w", err)
-	}
 	return nil
 }
```

Comment:
shouldn't `ValidateOptions` be merged with `func (o *Options) Validate()` ?

Reply:
Pomerium can be run as separate services with separate configs.